src/my_project/tagging/requester.py

```
@@ -205,6 +202,72 @@ def issue_get_request(self) -> APIIssueResults:
+            logger.warning("\n" + response.text)
+        data = response.json()
+        if data["error"] != "OK":
+            logger.warning("Error, please investigate")
+            # raise RuntimeError("Error, please investigate")
+        items = data["results"]
+        if data["number_of_page_results"] != 1:
+            raise RuntimeError("Error, please investigate")
+
+        return ComicVineDetailStruct.model_validate(items)
+
+    def get_publisher_info(self, volume_id: int) -> Publisher:
```

Ruff (0.16.1)

[warning] Unused method argument: `volume_id`

(ARG002)